src/utils/private.ts

```diff
@@ -118,24 +117,22 @@ export const getScalarKey = async (X25519_PRIVATE_KEY: string | undefined): Prom
 export const getWideConfig = async (context: Context) => {
   const orgConfig = await getConfigSuperset(context, "org");
   const repoConfig = await getConfigSuperset(context, "repo");
-  const defaultConfig = readFileSync("default_config.yml", "utf8");
 
-  const parsedOrg: WideOrgConfig | undefined = await parseYAML(orgConfig);
-  const parsedRepo: WideRepoConfig | undefined = await parseYAML(repoConfig);
+  const parsedOrg: WideOrgConfig | undefined = parseYAML(orgConfig);
+  const parsedRepo: WideRepoConfig | undefined = parseYAML(repoConfig);
```

Comment:
Is that correct to switch this to a synchronous function?

Reply:
yes, `YAML.parse` [does not return](https://eemeli.org/yaml/#yaml-parse) a promise